README.md

```diff
@@ -22,4 +22,4 @@ Inside the form element, you are presented with a textarea for additional commen
 Inside the form element, you are presented with a button with id of submit to submit all the inputs
 Fulfill the user stories and pass all the tests below to complete this project. Give it your own personal style. Happy Coding!
```

Comment:
- [ ]  Currently, your `README` file is not updated according to the project. Please update your `README` file because it is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. You can use [this template ](https://github.com/microverseinc/readme-template) for the `README` but remember to customise it to your project. :+1: